### ISSUE TITLE
Remove CircleCI dependency on Pontos

### ISSUE THREAD

**Issue by mcritchlow** [ucsdlib/damspas#481]
### Descriptive summary

Remove CircleCI test environment dependency on the pontos vm (which hosts damsrepo and solr in tomcat).

### Rationale

At the moment, we can only safely and reliably run one PR/branch test suite at a time because there is a singular dependency on pontos for acting as the repository and solr layer. 

This move to using Docker containers and the CircleCI container environment will allow multiple builds to run at the same time. 

NOTE: The builds are _still incredibly slow_. Recently our test suite has gone from about 45mins to taking twice that amount of time. It would be nice if we could both a) trim those new tests down to something more manageable and mock out what we can and b) run the test suite in parallel using the CircleCI ability to do that. I tried (b), but experienced a handful of test failures. As a result, that can happen, if we desire, in a separate effort.
